build/_nuget-resilient-publish.shade

```diff
@@ -0,0 +1,42 @@
+use namespace="System.Threading.Tasks"
+
+@{/*
+
+nuget-resilient-publish
+    Publishes NuGet packages with retries
+
+sourcePackagesDir=''
+    Required. Path to packages to push
+
+nugetFeed=''
+    Required. Feed to publish to.
+*/}
+
+@{
+   var packages = Directory.EnumerateFiles(sourcePackagesDir, "*.nupkg")
+                           .Where(p => !p.EndsWith(".symbols.nupkg"))
+                           .ToArray();
+                           
+    Parallel.ForEach(packages, package =>
+    {
+        var retries = 0;
+        var packagePath = Path.Combine(sourcePackagesDir, package);
+        start:
+        try
+        {
+            NuGetPush (nugetFeed, packagePath, "");
+        }
+        catch
+        {
+            if (++retries == 3)
```

Comment:
Maybe have retry count configurable?

- Also: There's no issues with publishing the same package to myget right? Just wondering in the case that we fail but a package was published.